contentcuration/contentcuration/static/js/bundle_modules/channel_edit.js

```diff
@@ -1,3 +1,6 @@
+// include all logic in "base" entrypoint
+require('./base');
```

Comment:
Join the 2 bundles. See comment [here](https://github.com/learningequality/studio/pull/582/files#diff-322f9cd0ebadbcc0da21764bcb8b0789R29)